Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -910,7 +913,7 @@ class BrowserViewController: UIViewController {
             
             // Ensure that the tab title *actually* changed to prevent repeated calls
             // to navigateInTab(tab:).
-            guard let title = tab.title else { break }
+            guard let title = (webView.title?.count == 0 ? webView.url?.absoluteString : webView.title) else { break }
```

Comment:
Brackets could be removed unless it feels more readable for you

Reply:
@kylehickinson

Reply:
I think I'd rather leave the brackets to be honest, looks more contained as a result to be unwrapped in the guard-let